custom_components/vikunja/config_flow.py

```diff
@@ -26,7 +27,8 @@ async def async_step_user(self, user_input=None) -> FlowResult:
             hide_done = user_input.get(CONF_HIDE_DONE, False)
             strict_ssl = user_input.get(CONF_STRICT_SSL, True)
 
-            api = VikunjaAPI(base_url, token, strict_ssl)
+            client = get_async_client(verify_ssl=strict_ssl)
```

Comment:
here also missing the `hass` argument. But I can't see a quick way to access the `hass` object so might need a bit more investigation.